specification/ai/ModelClient/models/chat_completions.tsp

```diff
@@ -0,0 +1,427 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+namespace ModelClient;
+
+@doc("""
+The configuration information for a chat completions request.
+Completions support a wide variety of tasks and generate text that continues from or "completes"
+provided prompt data.
+""")
+model ChatCompletionsOptions {
```

Comment:
```suggestion
alias ChatCompletionsOptions = {
```

Reply:
@johanste What is the advantage of defining this as an `alias` instead of a `model`? What is it wrong to do what I did? The emitted Python client library work fine as is, when this is defined as a `model`.

Reply:
It will/should give you keyword-only parameters. It avoids having to instantiate a "dummy" parameters instance.

Reply:
Or are you suggesting that in the emitted Python client library I don't have a class named ChatCompletionOptions as single argument to the operator method, but rather a have long list of method arguments, each one corresponding to a single setting? I thought the guidance was to group settings into an "Options" class. And, perhaps sperate the mandatory ones from the optional ones, which is something I want to discuss in SDK review. Only the "messages" property in the current ChatCompletionOptions is mandatory, and therefore the question I will have is should that be its own method argument on the operator, while the rest are left grouped in ChatCompletionOptions.

This is the Python SDK PR, free to leave comments there: https://github.com/Azure/azure-sdk-for-python/pull/34937

Reply:
Switching to using alias, following our phone call. Thanks!